vue/sbc-common-components/src/components/NotificationPanel.vue

```diff
@@ -0,0 +1,175 @@
+<template>
+  <div
+    v-if="showNotifications"
+    v-on:clickout="emitClose()"
+  >
+    <v-overlay></v-overlay>
+    <v-navigation-drawer
+      right
+      app
+      :width="440"
+      style="top: 70px; max-height: calc(100% - 0);"
```

Comment:
It looks like you have CSS for these in the style block below, so can you delete this property?

Reply:
The CSS one doesn't work. So I put it to give a try. I will look at it.

Reply:
If the v-navigation-drawer divs and classes are rendered at run-time, you may need to use `::v-deep` to select the right element.